packages/fxa-settings/src/components/ConnectedServices/Icon.tsx

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// https://codesandbox.io/s/react-dynamic-svg-import-ts-kzx7n?fontsize=14&hidenavigation=1&theme=dark
+import React, { useEffect, FC } from 'react';
+
+interface IconProps extends React.SVGProps<SVGSVGElement> {
+  name: string;
+}
+
+export const Icon: FC<IconProps> = ({ name, ...rest }): JSX.Element | null => {
+  const ImportedIconRef = React.useRef<
+    React.FC<React.SVGProps<SVGSVGElement>>
```

Comment:
nit: I suppose this could just be `FC` since it's imported and used on line 7.  Although I prefer the qualified name since `FC` is not a great name

Reply:
Switched so both instances use `React.FC`